src/gravel/controllers/orch/nfs.py

```diff
@@ -26,15 +28,33 @@ class NFSDaemonModel(BaseModel):
 
 
 class NFSServiceModel(BaseModel):
-    name: str
+    service_id: str
     daemons: List[NFSDaemonModel]
 
 
+class NFSBackingStoreEnum(str, Enum):
+    CEPHFS = 'cephfs'
+    RGW = 'rgw'
+
+
+class NFSExportModel(BaseModel):
+    export_id: int
+    path: str
+    pseudo: str
+    access_type: str
+    squash: str
+    security_label: bool
+    protocols: List[str]
+    transports: List[str]
+    fsal: Dict  # TODO: create model for this?
```

Comment:
nit: this is likely going to send pyright into throwing a fit because of missing arguments to `Dict`. `Dict[Any, Any]` would be preferable.